src/exporters/cds.py

```diff
@@ -241,7 +249,8 @@ def export(self,
                granularity: str = 'hourly',
                show_api_request: bool = True,
                selection_request: Optional[Dict] = None,
-               break_up: bool = True) -> List[Path]:
+               break_up: bool = True,
+               N_parallel_requests: int = 3) -> List[Path]:
```

Comment:
Nit: argument names should be lowercase. Can we rename this to `n_parallel_requests`?

Reply:
definitley